src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,128 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_ARRAY_SIZE = 10;
+    private int size;
+    private T[] listArray;
```

Comment:
```suggestion
    private T[] values;
```
naming's a bit confusing. Let's call it elements or values